his/core/application-integration-ladesigner-1.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Logic Apps Integration | Microsoft Docs"
+description: Extending Host Integration Server capabilities with Azure Logic Apps  
+author: haroldcampos
+ms.author: "hcampos"
+ms.prod: "host-integration-server"
+ms.service: #Required; use the name-string related to slug in ms.product/ms.service
+ms.topic: concept-article #Required; leave this attribute/value as-is.
+ms.date: "10/03/2023"
+
+#CustomerIntent: As a <type of user>, I want <what?> so that <why?>.
+---
+
+# Get started with the HIS Designer for Logic Apps
+
+Extending the Design of Integration scenarios to Azure Logic Apps by leveraging the core capabilities of the Host Integration Server TI Designer, with the new stand alone designer that removes the need of installing and configuring Host Integration Server for Azure Logic Apps based scenarios.
```

Comment:
Microsoft guidance for ["leverage"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/leverage)

```suggestion
You can extend the design of integration scenarios to Azure Logic Apps by using the core capabilities of the Host Integration Server TI Designer. This tool includes a new standalone designer that removes the need for installing and configuring Host Integration Server for Azure Logic Apps-based scenarios.
```